docs/2023-02-14-so-whats-next.md

```diff
@@ -211,45 +211,45 @@ The best way forward for JavaScript would be for TC39 and `core-js` to work toge
 
 ---
 
-After a while, "support" came from NPM. In `npm@7` which was released at the end of 2020 as a logical continuation of `npm fund` was disabled console output in post-install scripts. The result was expected - because people stopped seeing the funding request and almost no one uses `npm fund`, the number of `core-js` backers began to decline. Excellent support for the project from those who not only earn by distributing my work but also use it themselves -)
+After a while, "support" came from NPM. In `npm@7`, which was released at the end of 2020, as a logical continuation of `npm fund`, the console output was disabled in post-install scripts. The result was expectable, because people stopped seeing the funding request and almost no one uses `npm fund`, the number of `core-js` backers began to decline. An excellent support for the project from those, who not only earn by distributing my work, but also use it themselves -)
 
 <p align="center"><img alt="npm" width="720" src="https://user-images.githubusercontent.com/2213682/218333796-18bee93f-64e7-4257-8ddd-d16fc4f05989.png" /></p>
 
-In addition, another factor came into play again. Higher quality - less support. Is the library well-maintained? There are practically almost no open bug reports, and if they happen, is it fixed almost instantly? Does the library already give us almost everything we want? Yes? So why should we support the maintenance of the library? The price at which this is done for the maintainers is not on the surface - for the most, it's still just "a small library". Many of those who backed `core-js` before stopped doing it.
+In addition, another factor came into play again. Higher quality - less support. Is the library well-maintained? There are practically almost no open bug reports, and if they happen, is it fixed almost instantly? Does the library already give us almost everything we want? Yes? So why should we support the maintenance of the library? The price at which this is done for the maintainers is not on the surface - for most people, it's still just "a small library". Many of those who backed `core-js` before stopped doing it.
```

Comment:
I'm not sure about "people" since it's also companies. Maybe "users"?

Reply:
If you want to put emphasis on that, I would write it out as 'developers and companies', because to me 'users' psychologically means regular people, unless we started a thesaurus debate.

Reply:
"Developers and companies" sounds good.